source/includes/_assets.md

```diff
@@ -164,60 +168,43 @@ curl -X POST https://api.gresb.com/api/v1/entities/5028/assets \
   -H "Content-Type: application/json" \
   -d @- <<JSON
 {
-    "country": "US",
-    "state_province": "DC",
-    "city": "Washington, DC",
-    "address": "1600 Pennsylvania Avenue NW",
-    "construction_year": 1800,
-    "partners_id": "USGOV_DC456123G",
-    "certifications": [
-    {
-      "certification_id": 901,
-      "name": "Sunshine Energy A",
-      "level": "Premium",
-      "size": "230"          
-    },
-    {
-      "certification_id": 873,
-      "name": "Sunshine Energy A",
-      "level": "Elementary",
-      "size": "270"          
-    }
-    ],
+    "country": "CD",
+    "state_province": "flevoland",
+    "city": "franekeradeel",
+    "address": "ABC",
+    "partners_id": null,
+    "construction_year": null,
+    "asset_ownership": null,
+    "certifications": [],
+    "asset_size": null,
     "annual_data": [
         {
-            "year": 2020,
-            "asset_name": "The White House",
-            "whole_building": true,
-            "asset_size": 500,
-            "property_type_code": "OCHI",
-            "en_tot_lc_te": 147.12,
-            "wat_abs_lc_t": 97.1748
-        },
-        {
-            "year": 2019,
-            "asset_name": "The White House",
-            "asset_size": 500,
-            "property_type_code": "OCHI",
-            "asset_size": 500,
-            "en_tot_lc_te": 112.4,
-            "wat_abs_lc_t": 75.08
-        },
-        {
-            "year": 2018,
-            "asset_name": "The White House",
-            "asset_size": 500,
-            "property_type_code": "OCHI",
-            "en_tot_lc_te": 98.3,
-            "wat_abs_lc_t": 72.44
-        },
-        {
-            "year": 2017,
-            "asset_name": "The White House",
-            "asset_size": 500,
-            "property_type_code": "OCHI",
-            "en_tot_lc_te": 91.7,
-            "wat_abs_lc_t": 69.11
+            "year": 2021,
+            "asset_size": 21,
+            "asset_gav":1,
+            "asset_ownership":0,
+            "property_type_code": "OTSS",
+            "asset_name": "APITest2",
```

Comment:
use `APITest` for all instead of `APITest2`, `APITest3`, seems more professional that way.